frontends/pytorch/python/torch_mlir/torchscript/e2e_test/configs/ref_backend.py

```diff
@@ -26,9 +31,51 @@ def compile(self, program: torch.nn.Module) -> Any:
 
         extract_annotations(program, scripted, class_annotator)
 
-        mb.import_module(scripted._c, class_annotator)
-        # Lower module in place.
-        frontend_lowering.lower_object_graph(mb.module)
+        # TODO: Find a way to make each of these calls own its own
+        # "debuggable error report" situation.
+        try:
+            sys.stderr = StringIO()
```

Comment:
Is this a matter of actually having the MLIR diagnostic engine plumbed through Python? (because that has been on the list for a long time and been meaning to do it)

Reply:
(if this is going to be a long term thing, I would recommend some eliminating of redundancy in the way this is done)